user-interface/src/search-results/SearchResults.tsx

```diff
@@ -28,36 +20,73 @@ export function isValidSearchPredicate(searchPredicate: CasesSearchPredicate): b
   }, false);
 }
 
+export function sortByDateFiled(a: CaseBasics, b: CaseBasics): number {
+  if (a.dateFiled < b.dateFiled) {
+    return 1;
+  } else if (a.dateFiled > b.dateFiled) {
+    return -1;
+  } else {
+    return 0;
+  }
+}
+
+export function sortByCaseId(a: CaseBasics, b: CaseBasics): number {
+  if (a.caseId < b.caseId) {
+    return 1;
+  } else if (a.caseId > b.caseId) {
+    return -1;
+  } else {
+    return 0;
+  }
+}
+
+export function sortCaseList(a: CaseBasics, b: CaseBasics) {
+  // Date filed take priority over case Id.
```

Comment:
I'd rather we name the function something like `sortCaseListInPriorityOrder` or something than to have this comment.

Reply:
But are we even using this? I only see it called in a test.